config/config.go

```diff
@@ -174,12 +175,32 @@ format: "human"
 var _ yaml.Defaulter = &LoggingOutletEnumList{}
 
 type Global struct {
+	configDir string
+
 	Logging    *LoggingOutletEnumList `yaml:"logging,optional,fromdefaults"`
 	Monitoring []MonitoringEnum       `yaml:"monitoring,optional"`
 	Control    *GlobalControl         `yaml:"control,optional,fromdefaults"`
 	Serve      *GlobalServe           `yaml:"serve,optional,fromdefaults"`
 }
 
+func (g *Global) SetConfigDir(path string) error {
```

Comment:
Do you think always using full path names in the config hurts so much?
Relatiove to the config directory != working directory of the daemon => yet another relativ base path to keep track of...